src/region/__tests__/RegionAnnotation-test.tsx

```diff
@@ -11,15 +11,27 @@ describe('RegionAnnotation', () => {
         onSelect: jest.fn(),
         shape: rect,
     };
-    const getWrapper = (props = {}): ShallowWrapper => {
-        return shallow(<RegionAnnotation {...defaults} {...props} />);
+    const getWrapper = (props = {}): ReactWrapper => {
+        return mount(<RegionAnnotation {...defaults} {...props} />);
     };
 
     describe('render()', () => {
         test('should render the class name based on the isActive prop', () => {
-            expect(getWrapper().hasClass('ba-RegionAnnotation')).toBe(true);
-            expect(getWrapper({ isActive: true }).hasClass('is-active')).toBe(true);
-            expect(getWrapper({ isActive: false }).hasClass('is-active')).toBe(false);
+            expect(
+                getWrapper()
+                    .childAt(0)
```

Comment:
Does this need to be repeated for each block? What is it selecting? Moving it to a named a variable would help with both questions.

Reply:
yes I was planning to clean the tests up and add more. Since I am using useEffect I moved it to mount. Doesn't look like Enzyme is fully suited for all the hooks yet so still figuring out the best way to test this.